### PR TITLE
Add block part in sixth-section

### PR DESCRIPTION
**Pull request by Flasto4ka**
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -439,3 +439,35 @@ ul {
...
+.part__line {
+  width: 1169px;
+  height: 1px;
+
+  opacity: 0.2;
+  border: 1px solid #342b2b;
+}
+
+.part__title {
+  font-family: Roboto Slab, sans-serif;
```
Comment (flast1k) [inline css/style.css:454]: don't need. It's by default
By the way, you have two font-family properties for the body selector
image

File: index.html [hunk 2]
```diff
@@ -329,6 +345,43 @@ <h1 class="banner__title">
           </div>
         </div>
       </section>
+
+      <!-- Block part -->
+      <section class="part">
+        <div class="container">
+          <div class="row">
+            <div class="part__line"></div>
```
Comment (flast1k) [inline index.html:353]: for this purpose we have ht tag https://doka.guide/html/hr/

File: css/style.css [hunk 3]
```diff
@@ -439,3 +439,35 @@ ul {
   line-height: 20px;
   padding-right: 58px;
 }
+
+/* Block part */
+
+.part__line {
+  width: 1169px;
```
Comment (flast1k) [inline css/style.css:446]: just 100%

File: css/style.css (hunk 1 above)
Comment (Flasto4ka) [reply css/style.css:454]: +

File: css/style.css (hunk 3 above)
Comment (Flasto4ka) [reply css/style.css:446]: +

**Pull request merged**